decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,30 @@
+# Decision-making
+
+### Proposal submission
+Proposal is submitted by opening pull request to this repository.
+Initiator must present the idea on the architecture meeting.
+The announcement should include problem description the proposal is trying to resolve as well as solution(s).
+
+### Discussion
+There must be minimum 2 weeks of discussion in the pull request thread.
+After that initiator may decide when to start the vote.
+
+### Voting
+All eligible voters must get notification about the voting start.
+A valid voting period must be announced when voting is started, must be greater that 2 weeks and must not be changed during the vote.
+
+### Approval
+The number of `approve` votes must be greater than or equal to the number of `reject` votes multiplied by two in order to merge pull request.
+In case of multi-choice voting (implementation details) the decision may be taken by simple plurality. This means that the voting option with the most votes wins. If there are multiple options with the most number of votes, the initiator can choose one of them.
+Once pull request is merged the proposal is accepted.
+
+# Eligible voters
+The group of eligible voters should be established. The group should represent the Magento community and include internal developers and architects (40%), Magento maintainers (30%), extension developers and partner contributors (30%).
```

Comment:
> The group should represent the Magento community and include internal developers and architects (40%)

We already have architects assigned to the components.

Reply:
Yes, it's a known fact. How do you propose to use those assignments?

Reply:
Actually, it contradicts with the proposal: `The group should represent the Magento community and include internal developers and architects (40%)` because if all architects are included to the group, then the group will be too big otherwise the assigned architect might miss proposals for related components.